ui/src/main/js/common/component/table/TableHeader.js

```diff
@@ -27,7 +27,12 @@ const TableHeader = ({ columns, multiSelect, selected, onSelected, tableData  })
                 ) : null }
                 
                 { columns.map(column => (
-                    <TableHeaderCell key={column.key} label={column.label} sortable={column.sortable} settings={column.settings} />
+                    <TableHeaderCell 
```

Comment:
Can we add propTypes and defaults here for the properties used by this component.